evalml/tests/component_tests/test_utils.py

```diff
@@ -60,6 +60,9 @@ def test_scikit_learn_wrapper(X_y_binary, X_y_multi, X_y_regression):
                 num_classes = 3
             elif problem_type == ProblemTypes.REGRESSION:
                 X, y = X_y_regression
+            elif problem_type == ProblemTypes.TIME_SERIES_REGRESSION:
+                # Skipping because make_pipeline_from_components does not yet work for time series.
```

Comment:
The reason is because of the pipeline-level paramters of gap and max_delay. I imagine we'll use the same pattern to specify them that we will introduce in #1382 .